docs/troubleshoot/troubleshooting.md

```diff
@@ -2,6 +2,29 @@
 
 To isolate and resolve Zowe&trade; problems, you can use the troubleshooting and support information.
 
+## How to start troubleshooting
+
+When you run into some issues and look for troubleshooting tips, the following steps may help you.
+
+1. Search the error message or error code that you have in log on [Zowe doc site](https://docs.zowe.org/). If there are existing solutions, then you could follow them.
+
+2. If no solution is available or the existing solutions cannot apply to your problems, you could search the keywords, error messages, or error code in [Zowe GitHub repository](https://github.com/zowe). If the issues or pull requests are closed with solutions, you could follow them. If the issues are still open, you could leave your question or have a discussion in the thread to solve your issue.
+
+![A gif to show how to search in github repo](../images/troubleshoot/how-to-troubleshooting.gif)
+
+3. If your problems are not solved, you could try the following options:
```

Comment:
Instead of "problems", I'd use the singular: "problem."

Reply:
Also suggesting: Be more direct in your instructions: "... not solved, try the following options:"